internal/impl/amqp1/output.go

```diff
@@ -180,7 +190,11 @@ func (a *amqp1Writer) Connect(ctx context.Context) (err error) {
 	}
 
 	// Create a sender
-	if sender, err = session.NewSender(ctx, a.targetAddr, nil); err != nil {
+	var opts *amqp.SenderOptions
+	if a.persistent {
+		opts = &amqp.SenderOptions{SettlementMode: amqp.SenderSettleModeUnsettled.Ptr()}
```

Comment:
The default value for the SettlementMode is "MixedMode":

```
	// Sender MAY send a mixture of settled and unsettled deliveries to the receiver.
	SenderSettleModeMixed SenderSettleMode = encoding.SenderSettleModeMixed
```
https://github.com/Azure/go-amqp/blob/1003610b1e679a884a60f9011c37e55c6eb07338/const.go#L20

Changing the mode to "Unsettled" could be unnecessary.